### PR TITLE
use Dropdown in TaskCardActions

### PR DESCRIPTION
**Pull request by drewfead**
solves #17.

also adds aurelia dependencies.

### CODE REVIEW

File: src/task-card/TaskCardActions.js [hunk 1]
```diff
@@ -1,87 +1,54 @@
...
-                menuOptions.push(<li key={`header:${key}`} className="dropdown-header">{key}</li>);
-                options[key].forEach(item => {
-                    menuOptions.push(<li key={`${key}:${item.label}`}><a onClick={this.closeAnd(item.callback)}>{item.label}</a></li>);
-                });
+        const options = [
+            {
+                name: 'move to Todo',
+                items: [
+                    {
+                        key : props.isDone ? 'Re Open' : 'Unassign',
```
Comment (jhoguet) [inline src/task-card/TaskCardActions.js]: would prefer something like 'move-to-todo' for the key... mostly that way the cognitive overhead of tertiary for the label, is limited to the label

File: src/task-card/TaskCardActions.js [hunk 2]
```diff
@@ -1,87 +1,54 @@
...
-                <button className="btn btn-default dropdown-toggle" type="button" onClick={this.toggleOpen} ref="button">
-                    <span className="fa fa-cog" />
-                </button>
-                <ul className={`dropdown-menu`} ref="dropdown">
-                    {menuOptions}
-                </ul>
+            <div className="task-actions-wrapper">
+                <Dropdown
+                    buttonTemplate={ () => ( <ActionsIcon /> )}
+                    itemGroups={ options.filter(option => option.items.length) }
```
Comment (jhoguet) [inline src/task-card/TaskCardActions.js]: nit pick - but I would rather see `=> option.items.length > 0` here as it is more intuitive what the intent is... granted this is a filter... so obviously the intent is to filter and therefore length should be cast to boolean

File: src/task-card/TaskCardActions.js [hunk 3]
```diff
@@ -1,87 +1,54 @@
...
+                    itemTemplate={ item => item.key }
+                    onSelect={ item => ( item.callback() )}
```
Comment (jhoguet) [inline src/task-card/TaskCardActions.js]: the wrapping parens is unnecessary and therefore distracting (it is often preferred when wrapping some JSX - but this isn't JSX)

**Comment by drewfead** [comment]
alright @jhoguet--incorporated your feedback

**Pull request merged**